jest.config.js

```diff
@@ -14,6 +14,7 @@ module.exports = {
     'android.jsx',
     'android.js',
   ],
+  testEnvironment: process.env.RUN_ALLURE === true ? 'allure-jest/node' : 'jsdom',
```

Comment:
il me semble que les variables environnements sont toujours des `string` (peut etre parfois des `number`)

```suggestion
  testEnvironment: process.env.RUN_ALLURE === 'true' ? 'allure-jest/node' : 'jsdom',
```